### PR TITLE
Slo blog

### PR DESCRIPTION
**Pull request by hayleycd**
SLO blog. Links to FAQ

### CODE REVIEW

File: gcp/appengine/blog/content/posts/announcing-osv-service-level-objectives.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+We are excited to announce that OSV has published our new service level objectives (SLOs). 
+<!--more-->
+
+Even though OSV.dev is a free resource, we know that our users rely on us for accurate, up-to-date vulnerability information and we want to provide that on a consistent basis. We will be tracking the following:  
+
+1. Website and API availability, measured across a seven day rolling window.   
+2. Website and API latency.   
+3. Data freshness.  
+
+Please refer to our [FAQ](https://osv.dev/about) for specific details.
```
Comment (andrewpollock) [inline gcp/appengine/blog/content/posts/announcing-osv-service-level-objectives.md:17]: Is it possible to deep link to the relevant FAQ entry?

File: gcp/appengine/blog/content/posts/announcing-osv-service-level-objectives.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Even though OSV.dev is a free resource, we know that our users rely on us for accurate, up-to-date vulnerability information and we want to provide that on a consistent basis. We will be tracking the following:  
+
+1. Website and API availability, measured across a seven day rolling window.   
+2. Website and API latency.   
+3. Data freshness.  
+
+Please refer to our [FAQ](https://osv.dev/about) for specific details.  
+
+Our performance will be shared via a quarterly blog post, with the hopes for a reporting dashboard in the future.
```
Comment (andrewpollock) [inline gcp/appengine/blog/content/posts/announcing-osv-service-level-objectives.md:19]:
```suggestion
Performance against the SLO will be shared via a quarterly blog post, with the intention of a reporting dashboard in the future.  
```

Comment (andrewpollock) [reply gcp/appengine/blog/content/posts/announcing-osv-service-level-objectives.md:19]: Couple of minor tweaks

**Review by andrewpollock — COMMENTED** [review]
Close to LGTM

File: gcp/appengine/blog/content/posts/announcing-osv-service-level-objectives.md (hunk 1 above)
Comment (andrewpollock) [reply gcp/appengine/blog/content/posts/announcing-osv-service-level-objectives.md:17]: @oliverchang I think this is something to consider, are GitHub Discussions more fit-for-purpose as a dynamic FAQ than the FAQ as it currently exists on osv.dev?

**Review by andrewpollock — APPROVED** [review]

File: gcp/appengine/blog/content/posts/announcing-osv-service-level-objectives.md (hunk 1 above)
Comment (hayleycd) [reply gcp/appengine/blog/content/posts/announcing-osv-service-level-objectives.md:17]: @andrewpollock I am in the process of reworking the docs, happy to meet to discuss. I think that the FAQ could find a new home in the docs and we could link to headings there. Open to other options too.

File: gcp/appengine/blog/content/posts/announcing-osv-service-level-objectives.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: "Announcing OSV's Service Level Objectives"
+date: 2023-03-20T22:00:00Z
```
Comment (hayleycd) [inline gcp/appengine/blog/content/posts/announcing-osv-service-level-objectives.md:3]:
```suggestion
date: 2023-03-27T22:00:00Z
```

**Comment by hayleycd** [comment]
@andrewpollock I updated the date to reflect this going out in a release next week.

**Pull request merged**